packages/data-point/lib/middleware/factory.js

```diff
@@ -1,10 +0,0 @@
-function Middleware (name, callback) {
-  this.name = name
-  this.callback = callback
-}
-
-function create (name, callback) {
-  return new Middleware(name, callback)
-}
-
-module.exports.create = create
```

Comment:
The name property is no longer necessary because we don't need it to filter middleware from the `stack` array. Since `name` was removed, I decided to just use the callback function directly instead of wrapping it in the `Middleware` class